### ISSUE TITLE
Physical Buttons for navigation 

### ISSUE THREAD

**Issue by 0x414c49** [joBr99/nspanel-lovelace-ui#190]
I'm trying to find a workaround to use physical buttons as navigation buttons, for example, the left button always navigates to the first/home page.

In that case, wish we had the opportunity to disable the physical relays in the device from clicking.

**Comment by 0x414c49** [comment]
Gonna try it now.

**Comment by 0x414c49** [comment]
How can I do it with navigation?  I tried to read the source code and all docs but couldn't understand how to do it?

```
{"CustomRecv":"event,buttonPress2,hwbtn,bNavigate"}
```

How to pass the page name with it?

**Comment by 0x414c49** [comment]
Thanks, I almost learned everything about apps.yaml. and all my pages have a key.

**Comment by 0x414c49** [comment]
Tested it and it works fine.

**Comment by Lux-WorldPC** [comment]
@0x414c49  hi, is it possible to have the complete working code example for apps.yaml ?

**Comment by joBr99** [comment]
@0x414c49 Feel free to send a PR to add your config (or an anonymized version of it) to the appdeamon folder of this repo, would be nice to have some more advanced config examples.

**Comment by qnpns** [comment]
Hello, in case it can interrest someone who doesn't use the relays, I use the SetOption73 feature of Tasmota. 
With this option the buttons are detached from the relays and it allows to use multi-press.
After that it is very easy to create automations from the device page in HA : 

![NSP_multiclic](https://user-images.githubusercontent.com/82996464/168832938-7bf64b6a-dfc5-4d9e-afe4-6f8ead304f94.PNG)

For exemple I use the single clic on left button to go back to my menu page : 
```
alias: NSP Retour Menu Bouton gauche
description: ''
trigger:
  - platform: device
    domain: tasmota
    device_id: xxxx
    type: button_short_press
    subtype: button_1
    discovery_id: yyy_button_1_SINGLE
condition: []
action:
  - service: mqtt.publish
    data:
      topic: tele/nspanel1/RESULT
      payload: '{"CustomRecv":"event,buttonPress2,navigate.cardGrid_Menu,button"}'
mode: single
```

**Comment by Lux-WorldPC** [comment]
@qnpns top top top many. thanks.   working

**Comment by JaroslawPrzybyl** [comment]
Gents, why do not I see the sensors that correspond to the physical buttons on the device card in home assistant? I made decoupling by command SetOption73 1, so now I can see in HA two switches connected to relays, and I can switch them, but no representation for buttons. In console of tasmota I see, that every click is recognized:
![image](https://user-images.githubusercontent.com/33628620/194750698-dec44a96-11f6-4030-a087-758419013f5e.png)

but no reaction at all on HA side, even no such sensors, only temperature:
![image](https://user-images.githubusercontent.com/33628620/194750764-6ba414d3-8781-4807-a924-85fa41362710.png)

I already tried to remove device and add again with this new configuration, same result.
